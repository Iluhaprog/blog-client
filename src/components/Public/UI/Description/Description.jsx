import styled from 'styled-components';

const Description = styled.p`
  font-family: var(--public-font-2);
  font-size: 16px;
`;

export {Description};
