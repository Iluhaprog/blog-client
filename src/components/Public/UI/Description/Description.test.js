import {Description} from './Description';
import {render} from '@testing-library/react';

test('Should render Description component', () => {
  const {asFragment} = render(
    <Description>
      Test text
    </Description>
  );
  expect(asFragment()).toMatchSnapshot();
});
