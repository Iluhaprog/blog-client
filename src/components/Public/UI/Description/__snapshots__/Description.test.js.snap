// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Description component 1`] = `
<DocumentFragment>
  <p
    class="sc-bdfBwQ dKYnui"
  >
    Test text
  </p>
</DocumentFragment>
`;
